tests/integration/test_github.py

```diff
@@ -414,6 +413,70 @@ def test_update_pr_info(self):
         assert pr_info.title == orig_title
         assert pr_info.description == orig_description
 
+    def test_pr_create(self):
+        """
+        Requires  packit_service:test_source to be ahead of packit_service:test_target
+        at least by one commit.
+        """
+
+        def _pr_close(pr_id):
+            """
+            Temporary function, should be replaced by GithubProject.pr_close(), once implemented
+            """
+            pr = self.hello_world_project.github_repo.get_pull(int(pr_id))
+            pr.edit(state="closed")
+
+        pr_upstream_upstream = self.hello_world_project.pr_create(
+            title="upstream <- upstream",
+            body="pull request body",
+            target_branch="test_target",
+            source_branch="test_source",
+        )
+        _pr_close(pr_upstream_upstream.id)
+
+        pr_upstream_forkusername = self.hello_world_project.pr_create(
+            title="upstream <- fork_username:source_branch",
+            body="pull request body",
+            target_branch="test_target",
+            source_branch="test_source",
+            fork_username=self.hello_world_project.service.user.get_username(),
+        )
+        _pr_close(pr_upstream_forkusername.id)
+
+        pr_upstream_fork = self.hello_world_project.get_fork().pr_create(
+            title="upstream <- fork",
+            body="pull request body",
+            target_branch="test_target",
+            source_branch="test_source",
+        )
+        _pr_close(pr_upstream_fork.id)
+
+        pr_upstream_fork_fu_ignored = self.hello_world_project.get_fork().pr_create(
+            title="upstream <- fork (fork_username ignored)",
+            body="pull request body",
+            target_branch="test_target",
+            source_branch="test_source",
+            fork_username=self.hello_world_project.service.user.get_username(),
+        )
+        _pr_close(pr_upstream_fork_fu_ignored.id)
```

Comment:
I know it's not so easy to generate all these response files, but what do you think about splitting all these cases to multiple test methods?

Reply:
done